### PR TITLE
Support for Flight Create Orders & Order Management

### PR DESCRIPTION
**Pull request by tsolakoua**

### CODE REVIEW

**Comment by tsolakoua** [comment]
[[Python SDK] Flight Create Orders](https://trello.com/c/qLh8amYn/78-python-sdk-flight-create-orders)

File: README.rst [hunk 1]
```diff
@@ -301,6 +301,12 @@ List of supported endpoints
     amadeus.shopping.flight_offers.pricing.post(flights[0])
     amadeus.shopping.flight_offers.pricing.post(flights[0:2], include='credit-card-fees,other-services')
 
+    # Flight Create Orders
+    amadeus.booking.flight_orders.post(flights[0], traveler)
+
+    # Flight Order Management
+    amadeus.booking.flight_order('eJzTd9f3NjIJdzUGAAp%2fAiY=').get()
```
Comment (anthonyroux) [inline README.rst]: We have a problem with this API, the order id comes from flight booking and is temporary in test... So this example will work for a while then will stop working... Should we just explain that it comes from the previous request' id?

File: specs/namespaces/namespaces_spec.py [hunk 2]
```diff
@@ -306,3 +310,12 @@
                 {'data': {'type': 'flight-offers-pricing',
                           'flightOffers': [{'foo': 'bar'}]}}
             ))
+        with it('.shopping.booking.flight_orders.post'):
+            self.client.booking.flight_orders.post({'foo': 'bar'}, {'bar': 'foo'})
```
Comment (anthonyroux) [inline specs/namespaces/namespaces_spec.py]: do we miss the unit test for the flight order management get?

Comment (tsolakoua) [reply specs/namespaces/namespaces_spec.py]: The unit test is here https://github.com/amadeus4dev/amadeus-python/pull/62/files/ab18882346a41e2468aa7854316f8a3239c9cf80#diff-6c2042d33b9a79ce08554c558ee65e69R117

Comment (tsolakoua) [reply specs/namespaces/namespaces_spec.py]: ah right, there is one more missing. Adding it now

**Comment by tsolakoua** [comment]
> Looks fine by me. How is the code coverage report? 

The coverage is 98%

**Comment by anthonyroux** [comment]
> > Looks fine by me. How is the code coverage report?
> 
> The coverage is 98%

Did it go down a bit? Still very good but next time we work on it we can take a look at what we are missing :)

**Pull request merged**